app/ui/src/app/customizations/api-connector/api-connector.service.ts

```diff
@@ -4,13 +4,18 @@ import { Restangular } from 'ngx-restangular';
 
 import { RESTService } from '@syndesis/ui/store/entity';
 import { ApiConnectorData, ApiConnectors } from './api-connector.models';
+import { ApiHttpService } from '@syndesis/ui/platform';
 
 @Injectable()
 export class ApiConnectorService extends RESTService<ApiConnectorData, ApiConnectors> {
-  constructor(restangular: Restangular) {
+  constructor(restangular: Restangular, private apiHttpService: ApiHttpService) {
     super(restangular.service('connectors?query=connectorGroupId%3Dswagger-connector-template'), 'apiConnector');
   }
 
+  public getApiConnector(id): Observable<ApiConnectorData> {
```

Comment:
This is nice. Two suggestions:

- Class methods and properties are implicitly `public` by default in TypeScript, so you'll want to remove the `public` accessor (and never use it again except for some edge case scenarios).
- Although it doesn't fall in the scope of this task, you might want to replace the Restangular implementation in the `list()` method below by a more lean ApiHttpService implementation like:

```typescript
list(): Observable<ApiConnectors> {
  return this.apiHttpService
    .setEndpointUrl('connectors?query=connectorGroupId%3Dswagger-connector-template')
    .get< ApiConnectors >();
}
``` 

this obviously is not part of this issue but would allow to remove all the Restangular boilerplate from this service file at once. It's up to you ;)